src/libraries/System.Net.Sockets/tests/FunctionalTests/CreateSocketTests.cs

```diff
@@ -244,17 +245,28 @@ public void Ctor_Netcoreapp_Success(AddressFamily addressFamily)
             s.Close();
         }
 
-        [Fact]
-        public void Ctor_SafeHandle_Invalid_ThrowsException()
```

Comment:
Why was this test removed?

I get that the pipe-specific stuff may no longer be relevant, but it seems like (a) other parts of the test are still useful; (b) we could modify the pipe part to use a different invalid fd, like a file fd.

Reply:
I've added back the (a) part.

For (b), we're not doing anything to consider file fds invalid.

Reply:
Thanks. Re (b) I think you are saying: the test validated that we failed on pipes before, but now we don't fail on pipes, so the test is not relevant. Am I understanding correctly?

Reply:
Yes.